app/views/devise/unlocks/new.html.erb

```diff
@@ -1,16 +1,24 @@
-<h2>Resend unlock instructions</h2>
+<div class="row">
+  <div class="col-xs-8 col-sm-6 col-md-4 col-xs-offset-2 col-sm-offset-3 col-md-offset-4">
```

Comment:
Same here.  Be sure to re-test all these views after the refactoring in web, iPad, and iPhone, to ensure they are still appearing as you'd like them to.